dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -81,6 +81,48 @@ CWWKD1005.find.rtrn.err.explanation=The return type of the method is not \
 CWWKD1005.find.rtrn.err.useraction=Refer to the Jakarta Data Javadoc \
  information for a list of valid return types for repository find methods.
 
+CWWKD1006.delete.rtrn.err=CWWKD1006E: The {0} return type of the {1} method \
+ of the {2} repository interface is not a valid return type for a repository \
+ Delete operation. Use a return type of void to return no value. Use a return \
+ type of long or int for a deletion count. Use a return type of boolean for \
+ an indicator of whether any entities were deleted. Use a return type of array, \
+ List, or Optional, parameterized with either the {3} entity class or the \
+ {4} unique identifier class, to return the deleted entities or entity Ids.
+CWWKD1006.delete.rtrn.err.explanation=The return type of the method is not \
+ one of the supported return types for a delete operation.
+CWWKD1006.delete.rtrn.err.useraction=Update the repository method to use one \
+ of the supported return types.
+
+CWWKD1007.updel.rtrn.err=CWWKD1007E: The {0} return type of the {1} method \
+ of the {2} repository interface is not a valid return type for a repository \
+ {3} operation. Use a return type of void to return no value. Use a return \
+ type of long or int for a count of matching entities. Use a return type of \
+ boolean for an indicator of whether any entities match.
+CWWKD1007.updel.rtrn.err.explanation=The return type of the method is not \
+ one of the supported return types for an update or delete operation.
+CWWKD1007.updel.rtrn.err.useraction=Update the repository method to use one \
+ of the supported return types.
+
+CWWKD1008.ambig.rtrn.err=CWWKD1008E: The {0} return type of the {1} method \
```

Comment:
CWWKD1008.ambig.rtrn.err=CWWKD1008E: The {0} return type of the {1} method of the {2} repository interface is ambiguous as a return type because it corresponds to multiple entity properties: {3}. 

CWWKD1008.ambig.rtrn.err.explanation=The return type of the method is not one of the supported return types for a delete method. To use this result type, the repository method must switch to the Query annotation and use a JDQL or JPQL query with a SELECT clause that specifies which entity property to return.

CWWKD1008.ambig.rtrn.err.useraction=Update the repository method to use the Query annotation to select the entity property.